packages/react-frontend/src/components/QuestionForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { Box, Typography, Stack, Button, TextField } from "@mui/material";
+import { Prompt } from "./Prompt.js";
+import Feed from "./Feed.js";
+
+function QuestionForm(props){
+  
+
+  const [question, setQuestion] = useState({
+    question: ""
```

Comment:
There is no need for these hooks to be objects, they could all be string.
i.e.
const [question, setQuestion] = useState("");